cfme/intelligence/reports/schedules.py

```diff
@@ -27,6 +28,18 @@
 from widgetastic_manageiq import Table
 
 
+class VolatileBootstrapSelect(BootstrapSelect):
```

Comment:
Why not put this in `widgetastic_manageiq/__init__.py`? What a weird widget!

Reply:
It's a weird widget and will probably not be used anywhere else which is why I put it within the file.